docs/sample_config.yaml

```diff
@@ -1594,6 +1594,12 @@ room_prejoin_state:
 #
 #track_appservice_user_ips: true
 
+# Set to establish maximum backoff (in seconds) between HS -> AS connection attempts.
+# Upon failing to push appservice events the homeserver will wait
+# an increasing amount of seconds between retries. This sets an upper limit of that (in seconds)
+#
+#appservice_max_backoff: 60
```

Comment:
this doesn't seem to say what the default behaviour is?

Reply:
note that pretty much everything in the config file which is a duration is in milliseconds, so having a setting in seconds is confusing. Consider making it take a suffix like "s" for seconds, for consistency with all the other time period settings.